disassemblers/ofrak_ghidra/ofrak_ghidra_test/assets/example_ghidra_package/ghidra_scripts/GetCodeRegionsDuplicate.java

```diff
@@ -0,0 +1,88 @@
+import com.google.common.base.Strings;
+import ghidra.app.util.headless.HeadlessScript;
+import ghidra.app.script.GhidraState;
+import ghidra.program.model.address.Address;
+import ghidra.program.model.address.AddressRange;
+import ghidra.program.model.address.AddressRangeIterator;
+import ghidra.program.model.address.AddressSet;
+import ghidra.program.model.block.BasicBlockModel;
+import ghidra.program.model.block.CodeBlock;
+import ghidra.program.model.block.CodeBlockIterator;
+import ghidra.program.model.block.CodeBlockModel;
+import ghidra.program.model.listing.*;
+import ghidra.program.model.mem.Memory;
+import ghidra.program.model.mem.MemoryBlock;
+import ghidra.program.model.mem.MemoryBlockSourceInfo;
+import ghidra.program.model.pcode.PcodeOp;
+import ghidra.util.exception.CancelledException;
+import ghidra.util.task.TaskMonitor;
+import ghidra.program.model.lang.Register;
+import ghidra.program.model.lang.RegisterValue;
+import ghidra.program.model.symbol.Reference;
+import ghidra.program.model.block.CodeBlockReferenceIterator;
+import ghidra.program.model.symbol.ReferenceIterator;
+import ghidra.program.model.symbol.RefType;
+
+import java.io.IOException;
+import java.util.*;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import java.math.BigInteger;
+
+public class GetCodeRegionsDuplicate extends HeadlessScript {
+    @Override
+    public void run() throws Exception {
+        try{
+            String response = new GetCodeRegionsDuplicate.Result().toJson();
+            storeHeadlessValue("OfrakResult_GetCodeRegions", response);
+        } catch (Exception e){
+            println(e.toString());
```

Comment:
Does this `println` actually work? I remember that in Java you usually need `System.out.println`, but I don't see where that is imported?

Reply:
`println` is needed to actually write the message to the Ghidra logs, and is imported along with the rest of the standard Ghidra API implicitly when the Ghidra script is built. The normal java way to stdout seems to get swallowed by the void.